env/env.js

```diff
@@ -71,7 +71,7 @@ const development = removeNullValues({
   GOOGLE_STORAGE_BUCKET: 'vbm-dev-281822.appspot.com',
   DEV_SERVER_PORT: 8080,
   REACT_APP_DEFAULT_ADDRESS: 1,
-  REACT_APP_TIMEOUT: 2000,
+  REACT_APP_TIMEOUT: 10000,
```

Comment:
Everything is working on the server side, but this PR will break the client unless we do this change. Handling errors and waiting for every promise to complete takes about 5 seconds on `trpc.register` (I believe this duration will be shorter in the US).